### DISCUSSION TITLE
List of possible roles

### DISCUSSION THREAD

**Discussion by ddellarocca** [tchiotludo/akhq#1066] [Q&A] [answered]
Hi,
I can't find in the [docs](https://akhq.io/docs/configuration/) the list of possible roles that i can assign to groups, is there any?
In my case i have LDAP authentication and i map LDAP groups to AKHQ groups.

**Comment by eddyv** [comment]
i believe you can find it listed here: https://akhq.io/docs/configuration/authentifications/groups.html

**Reply by eddyv** [reply]
Any other roles you'll have to create yourself

**Reply by ddellarocca** [reply]
That's not what I'm looking for, i mean the possible roles as used here https://github.com/tchiotludo/akhq/blob/af3ecb8f9d98b919a3fb06ea109135a69854fdf4/application.example.yml#L188

**Comment by tchiotludo** [comment]
@ddellarocca: I really understand what you want if not the roles like `topic/read` ?

**Reply by ddellarocca** [reply]
I want to know if there is a list of all possible roles that i can use to compose a custom group

**Comment by tchiotludo** [comment, marked as answer]
It's not role that you want I think but it's groups. 
You have 3 default groups [here](https://github.com/tchiotludo/akhq/blob/dev/src/main/resources/application.yml#L133) but you can't add as many as you want.

**Reply by ddellarocca** [reply]
The 3 default groups don't fit in my need, I want to create my own and I'm looking for a list of all possible roles that I can add to my groups

**Reply by tchiotludo** [reply]
```yaml
        - topic/read
        - topic/insert
        - topic/delete
        - topic/config/update
        - node/read
        - node/config/update
        - topic/data/read
        - topic/data/insert
        - topic/data/delete
        - group/read
        - group/delete
        - group/offsets/update
        - registry/read
        - registry/insert
        - registry/update
        - registry/delete
        - registry/version/delete
        - acls/read
        - connect/read
        - connect/insert
        - connect/update
        - connect/delete
        - connect/state/update
```
